frontend/src/scenes/session-recordings/player/snapshotDataLogic.tsx

```diff
@@ -24,6 +24,8 @@ import {
 import type { snapshotDataLogicType } from './snapshotDataLogicType'
 
 const DEFAULT_V2_POLLING_INTERVAL_MS = 10000
+const MAX_V2_POLLING_INTERVAL_MS = 60000
+const POLLING_INACTIVITY_TIMEOUT_MS = 5 * MAX_V2_POLLING_INTERVAL_MS
```

Comment:
since i'm changing it anyway... currently a page that was polling would simply sit and poll for sources forever... boooo